extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -535,10 +535,11 @@ export class AmpStoryPage extends AMP.BaseElement {
         this.preloadAllMedia_().then(() => {
           this.startMeasuringAllVideoPerformance_();
           this.startListeningToVideoEvents_();
-          this.playAllMedia_();
-          if (!this.storeService_.get(StateProperty.MUTED_STATE)) {
-            this.unmuteAllMedia();
-          }
+          this.playAllMedia_().then(() => {
```

Comment:
Please add a small comment like
"iOS 14.* requires play to be called before unmute...."